src/__screenshot_tests__/skeletons-screenshot-test.tsx

```diff
@@ -0,0 +1,89 @@
+import {openStoryPage, screen} from '../test-utils';
+
+test.each`
+    givenStoryPageArgs
+    ${{inverse: false, disableAnimation: true}}
+    ${{inverse: true, disableAnimation: true}}
+    ${{inverse: false, size: 60, disableAnimation: true}}
+    ${{inverse: true, size: 60, disableAnimation: true}}
+    ${{inverse: false, size: 100, disableAnimation: true}}
+    ${{inverse: true, size: 100, disableAnimation: true}}
+`(
+    `Skeleton circle given inverse $givenStoryPageArgs.inverse and size of $givenStoryPageArgs.size`,
+    async ({givenStoryPageArgs}) => {
+        await openStoryPage({
+            id: 'components-skeletons-skeleton-circle',
+            device: 'MOBILE_IOS',
+            args: givenStoryPageArgs,
+        });
+
+        const element = await screen.findByTestId('skeleton-circle');
+        expect(await element.screenshot()).toMatchImageSnapshot();
+    }
+);
```

Comment:
I'd rewrite these tests as:
```
test.each`
    inverse  | size
    ${false} | ${undefined}
    ${true}  | ${undefined}
    ${false} | ${60}
    ${true}  | ${60}
    ...
`(
    `Skeleton circle given inverse $inverse and size of $size`,
    async ({givenStoryPageArgs}) => {
        await openStoryPage({
            id: 'components-skeletons-skeleton-circle',
            device: 'MOBILE_IOS',
            args: {inverse, size, disableAnimation: true},
        });

        const element = await screen.findByTestId('skeleton-circle');
        expect(await element.screenshot()).toMatchImageSnapshot();
    }
);
```

Reply:
there's some issue with the undefined, which does not make the element render properly for the test. this only happens within the test env, the components themselves accept undefined without any problem. that's why I've used the objet in order to control the undefined values as a missing ptop